metrics_utility/automation_controller_billing/dataframe_engine/dataframe_jobhost_summary_usage.py

```diff
@@ -2,12 +2,20 @@
 from debug_utils import print_data, print_debug
 
 import pandas as pd
+import os
 
 from metrics_utility.automation_controller_billing.dataframe_engine.base import \
     Base
 
 logger = logging.getLogger(__name__)
 
+DIRECT = 0
+INDIRECT = 1
+# EDGE = 2
+
+def get_optional_collectors():
+    return os.environ.get('METRICS_UTILITY_OPTIONAL_COLLECTORS', 'main_jobevent').split(",")
```

Comment:
try to pass all the env vars via https://github.com/ansible/metrics-utility/blob/devel/metrics_utility/management/commands/build_report.py#L179

So it's easier to manage the config.

Reply:
I included it here, but I also have helper file with coputing directly variable INCLUDE_INDIRECT so I can easily to import it anywhere, because it is also used in collectors and not only in build_report.